### PR TITLE
Fix issues with Geocodio responses without streets

### PR DESCRIPTION
**Pull request by wilwade**
Handle Geocodio responses that do not include a street, number, or suffix fields. Issue #20

### CODE REVIEW

File: src/Geocoder/Provider/GeocodioProvider.php [hunk 1]
```diff
@@ -126,7 +126,19 @@ protected function executeQuery($query)
 
                 $coordinates = $location['location'];
                 $address = $location['address_components'];
-                $street = $address['street'] ?: null;
+
+				//Geocodio does not always return a street, number, or suffix
+                if (!isset($address['street']) && isset($json['input']['address_components']['street'])) {
+					//Sometimes Geocodio returns parsed information in the input
```
Comment (toin0u) [inline src/Geocoder/Provider/GeocodioProvider.php]: I seems something went wrong with indentations

**Comment by toin0u** [comment]
It looks good to me otherwise :)

**Comment by wilwade** [comment]
Fixed that formatting. Sorry about that, I forgot to double check in vim. Squashed in.

**Comment by toin0u** [comment]
No problem! Thanks for fixing this!

@willdurand I think this can be merged :)

**Comment by willdurand** [comment]
thx

**Pull request merged**